### DISCUSSION TITLE
Loco positioning system - anchor position setup

### DISCUSSION THREAD

**Discussion by frulloa** [bitcraze/discussions#655] [Q&A] [answered]
Hello. Please kindly help.

For the anchor's position setup. What do I have to consider as a 0,0,0 coordinate?
Maybe the center of my workspace or can be an anchor itself?
In the picture of the tutorial, it looks like you consider the center of the workspace as 0,0,0 
![Screenshot from 2023-04-12 12-18-32](https://user-images.githubusercontent.com/78211872/231429830-f4538f6f-2023-41a2-ad31-0c0916d20c34.png)

**Comment by krichardsson** [comment, marked as answer]
It is not that important where the origin is located, but there are a couple of things to keep in mind.

1. The default behavior of the code is generally to assume that the Crazyflie is "close" to the origin when turned on. For instance, the kalman estimator is reset to (0, 0, 0), and will usually converge to the correct position as long the real CF position is not too far away (within a few meters). If you configure your origin to be something like 20 meters away, this might not work and you will have to configure the reset position to be close to the real position. 
2. There is a general assumption that the floor is at Z=0, some scripts will go to Z=0 when landing for instance

**Reply by frulloa** [reply]
Thank you!
